### ISSUE TITLE
Need dispatch version command

### ISSUE THREAD

**Issue by markpeek** [vmware/dispatch#144]
- [ ] Bug
- [X] Feature
- [ ] Enhancement

## Detailed Description
Ran the following command and expected to see what version was installed.

```
$ dispatch version
Error: unknown command "version" for "dispatch"
Run 'dispatch --help' for usage.
```

## Context
It would be good to know what version is installed and whether to upgrade to a more current version.

**Comment by kars7e** [comment]
Partially addressed by #289 (still needs server side version)

**Comment by berndtj** [comment]
Sorry, I didn't see that last comment when I made my review comments

**Comment by imikushin** [comment]
Addressed by #500
